packages/app/components/jsonschemaform/widgets/AnchorWidget.tsx

```diff
@@ -0,0 +1,44 @@
+import type { WidgetProps } from '@rjsf/utils'
+import { useEffect, useState } from 'react'
+import { findAndReplace, macros } from './utils'
+
+/**
+ * Schema Layout:
+    "MyProperty": {
+        "ui:widget": "anchor",
+        "ui:options": {
+            "href": "http://localhost:8080/service-request-from-config?variable_entry_id=******",
```

Comment:
This is a cool solution, though I'd be interested in your thought process for using this vs. a named template like Handlebars

Reply:
Haha, honestly, more of a hindsight is 20/20 moment. I think Mustache / Handlebars would have worked great here! Even just using a named template would probably be an improvement. We could add that under another field (like `every`, but maybe `named`...but then maybe we'd need to make the `change` an array, since `every` => `to` wouldn't have the right relationship).